bootstrapvz/providers/ec2/__init__.py

```diff
@@ -80,12 +80,13 @@ def resolve_tasks(taskset, manifest):
         taskset.add(tasks.network.EnableDHCPCDDNS)
 
     if manifest.release >= jessie:
-        taskset.add(apt.AddBackports)
         taskset.add(tasks.tuning.SetCloudInitMountOptions)
         taskset.add(tasks.packages.AddWorkaroundGrowpart)
         taskset.add(initd.AdjustGrowpartWorkaround)
         if manifest.system['bootloader'] == 'grub':
             taskset.add(grub.EnableSystemd)
+        if manifest.release == stable:
```

Comment:
Ah, ok. The reason backports was added is actually in order to get `cloud-init` working, I think. So you could actually just make it `manifest.release == jessie`, it was added in #256, where there are a slew of other options added, so the justification for adding backports in general seems a little thin.